api/scpca_portal/models/project.py

```diff
@@ -843,7 +843,7 @@ def get_sample_input_data_dir(self, sample_scpca_id):
         """Returns an input data directory based on a sample ID."""
         return self.input_data_path / sample_scpca_id
 
-    def get_samples(
+    def create_samples(
```

Comment:
While we are here we should do the following:
- this seems like it would be better served as a class method on the Sample model
- I think we can improve the docstring on this method
- since this function essentially just wraps the `get_from_dict` and `bulk_create` methods, we should probably rename this to `bulk_create_from_dicts`